appinventor/appengine/src/com/google/appinventor/client/explorer/commands/DeleteFileCommand.java

```diff
@@ -36,6 +70,158 @@ public boolean willCallExecuteNextCommand() {
 
   @Override
   public void execute(final ProjectNode node) {
+	  if (node instanceof YoungAndroidSourceNode) {
+  		new DeleteFormDialog((YoungAndroidSourceNode) node).center();
+  		
+	  } 
+	  else {
+  		  executionFailedOrCanceled();
+  		  throw new IllegalArgumentException("node must be a YoungAndroidProjectNode");
+	  }
+  }
```

Comment:
Strange whitespace here. Our style guide calls for two spaces for each of these indents.

Reply:
Removed whitespace